app/controllers/users_controller.rb

```diff
@@ -12,10 +12,10 @@ def new
   def create
     begin
       create_paid_user
-    rescue Stripe::CardError => e
+    rescue StripeWrapper::CardError => e
       flash[:danger] = e.message
       render :new
-    rescue
+    rescue ActiveRecord::RecordInvalid => e
       flash.delete(:danger)
```

Comment:
Can I ask why you don't want an error displayed when there was a definite issue saving the record?